lib/kwtool/crawlers.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Kwtool.Crawlers do
+  import Ecto.Query, warn: false
+
+  alias Kwtool.Accounts.Schemas.User
+  alias Kwtool.Crawlers.Schemas.Keyword
+  alias Kwtool.Crawlers.UploadParser
+  alias Kwtool.Repo
+
+  def save_keywords_list(keyword_file, %User{} = user) do
+    keyword_file
+    |> UploadParser.load_from_file()
+    |> Enum.each(fn keyword ->
+      create_keyword(%{
+        phrase: List.first(keyword),
+        status: 0,
+        user_id: user.id
+      })
+    end)
+
+    {:ok, "The keyword file is processed successfully!"}
```

Comment:
This kind of success messages should be implemented at the controller level as it's an HTTP/Web layer concern. At the context level, returning `{:ok, created_keyword}` would be expected at the domain level.

Reply:
yes, i updated all other places as well.